deepspeed/checkpoint/constants.py

```diff
@@ -11,15 +11,30 @@
 
 BASE_OPTIMIZER_STATE = 'base_optimizer_state'
 SINGLE_PARTITION_OF_FP32_GROUPS = "single_partition_of_fp32_groups"
-GROUPS_PADDING = 'groups_padding'
-
+GROUP_PADDINGS = 'group_paddings'
 PARTITION_COUNT = 'partition_count'
 ZERO_STAGE = 'zero_stage'
 CLIP_GRAD = 'clip_grad'
+PARAM_SLICE_MAPPINGS = 'param_slice_mappings'
+FP32_WEIGHT_KEY = "fp32"
 
 #########################################
 # Module checkpoint keys
 #########################################
 PARAM_SHAPES = 'param_shapes'
 BUFFER_NAMES = 'buffer_names'
+
+#########################################
+# Checkpoint naming constants
+#########################################
+MODEL_FILE_PREFIX = 'mp_rank_'
+ZERO_FILE_PREFIX = 'bf16_' + 'zero_pp_rank_'
+OPTIM_FILE_SUFFIX = '_optim_states.pt'
+MODEL_FILE_SUFFIX = '_model_states.pt'
+LAYER_FILE_PREFIX = 'layer_'
+BF16_ZERO_FILE_PREFIX = ZERO_FILE_PREFIX
```

Comment:
Can we not support fp16 for ckpt reshaping?
Should be `ZERO_FILE_PREFIX = 'zero_pp_rank_'` in that case

Reply:
Apologies, this was a hack that I forgot to clean up. Yes, the goal is to support for fp16 and bf16. Will cleanup asap.

Reply:
Do you have an estimate when there will be fp16 support?

Reply:
I was away and now Tunji is away this week.

Given the lack of resources this is need driven, so first need to complete the bf16 work and then can look at fp16 next if there is a reasonable need for it and of course I can't speak on Tunji's behalf of his availability and priorities.